packages/controlled-form/src/lib/Select.stories.tsx

```diff
@@ -1,133 +1,94 @@
-        <FormControl>
-          <FormLabel id={`${args.id}-label`}>{args.label}</FormLabel>
-          <ControlledSelect {...args} labelId={`${args.id}-label`}>
-            <MenuItem value={1}>Option 1</MenuItem>
-            <MenuItem value={2}>Option 2</MenuItem>
-            <MenuItem value={3}>Option 3</MenuItem>
-          </ControlledSelect>
-          <Actions />
-          <SubmittedValues />
-        </FormControl>
-      </ControlledForm>
+      <FormProvider {...methods}>
+        <form onSubmit={methods.handleSubmit((data) => data)}>
+          <FormControl>
+            <FormLabel id={`${args.id}-label`}>{args.label}</FormLabel>
+            <ControlledSelect {...args} labelId={`${args.id}-label`}>
+              <MenuItem value={1}>Option 1</MenuItem>
+              <MenuItem value={2}>Option 2</MenuItem>
+              <MenuItem value={3}>Option 3</MenuItem>
+            </ControlledSelect>
+          </FormControl>
+          <Grid container direction="row" justifyContent="space-between" marginTop={1}>
+            <Button disabled={!methods?.formState?.isSubmitSuccessful} children="Reset" color="secondary" onClick={() => methods.reset()} />
+            <Button type="submit" disabled={methods?.formState?.isSubmitSuccessful} children="Submit" />
+          </Grid>
+          { methods?.formState?.isSubmitSuccessful ? (
+            <Paper sx={{ padding: '1.5rem', marginTop: '1.5rem' }}>
+              <Typography variant="h2">Submitted Values</Typography>
+              <pre data-testid="result">{JSON.stringify(methods.getValues(), null, 2)}</pre>
+            </Paper>
+          ) : null }
+        </form>
+      </FormProvider>
     );
   },
   args: {
     name: 'controlledSelect',
-    required: 'This is required.',
+    required: true,
+    rules: { required:'This is required.' },
     label: 'Select Label',
   },
 };
 
 export const _ControlledMultiSelect: StoryObj<typeof ControlledSelect> = {
   render: (args) => {
-    const SubmittedValues = () => {
-      const {
-        getValues,
-        formState: { isSubmitSuccessful },
-      } = useFormContext();
-
-      return isSubmitSuccessful ? (
-        <Paper sx={{ padding: '1.5rem', marginTop: '1.5rem' }}>
-          <Typography variant="h2">Submitted Values</Typography>
-          <pre>{JSON.stringify(getValues(), null, 2)}</pre>
-        </Paper>
-      ) : null;
-    };
-
-    const Actions = () => {
-      const {
-        reset,
-        formState: { isSubmitSuccessful },
-      } = useFormContext();
-      return (
-        <Grid container direction="row" justifyContent="space-between" marginTop={1}>
-          <Button
-            disabled={!isSubmitSuccessful}
-            children="Reset"
-            color="secondary"
-            onClick={() => reset({ [args.name]: [] })}
-          />
-          <Button type="submit" disabled={isSubmitSuccessful} children="Submit" />
-        </Grid>
-      );
-    };
+    const methods = useForm();
```

Comment:
value will be fixed in next commit. hoping to roll any other doc issues into one commit.